torch_harmonics/examples/models/sfno.py

```diff
@@ -53,10 +55,11 @@ def __init__(
         drop_rate=0.0,
         drop_path=0.0,
         act_layer=nn.GELU,
-        norm_layer=nn.Identity,
+        norm_layer="none",
```

Comment:
we are changing the defaults here, is that OK with respect to the examples?

Reply:
yes that is ok - none maps to identity